strcoll() does not exist in mes libc, change it to strcmp.

--- src/ls.c
+++ src/ls.c
@@ -2597,7 +2597,7 @@ xstrcoll (char const *a, char const *b)
 {
   int diff;
   errno = 0;
-  diff = strcoll (a, b);
+  diff = strcmp (a, b);
   if (errno)
     {
       error (0, errno, _("cannot compare file names %s and %s"),
